package.json

```diff
@@ -1,16 +1,21 @@
 {
   "name": "@yext/chat-ui-react",
-  "version": "0.8.2",
+  "version": "0.8.3",
   "description": "A library of React Components for powering Yext Chat integrations.",
   "author": "clippy@yext.com",
   "main": "./lib/commonjs/src/index.js",
   "module": "./lib/esm/src/index.mjs",
-  "types": "./lib/esm/src/index.d.ts",
   "sideEffects": false,
   "exports": {
     ".": {
-      "import": "./lib/esm/src/index.mjs",
-      "require": "./lib/commonjs/src/index.js"
+      "import": {
```

Comment:
not sure if I completely follow but it seems like exports wouldn't/shouldn't fix the core issue based on this comment? https://github.com/microsoft/TypeScript/issues/52363#issuecomment-1659227177

have you check if updating the extension of the files work? aka `index.d.cts` and` index.d.mts` for commonjs and esm respectively

Reply:
I don't actually know how to do this :(

Reply:
discussed on Slack. We can leave this out for now as it seems like tsc doesn't support this right now